<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallDialog</class>
 <widget class="QDialog" name="InstallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="m_ProgressHint">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>560</y>
     <width>731</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="m_ProgressBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>670</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="m_logo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>941</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_minuteLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>730</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_middle">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>730</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_secondLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>730</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_linuxVersionCombo">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>630</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_mountLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>770</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_objectLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>770</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="m_diskCapacityCombo">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>630</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_OSVersion">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_StartBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>740</y>
     <width>372</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_StopBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>750</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
